src/Agent.Sdk/ProcessInvoker.cs

```diff
@@ -189,7 +189,8 @@ public ProcessInvoker(ITraceWriter trace)
                 redirectStandardIn: redirectStandardIn,
                 inheritConsoleHandler: inheritConsoleHandler,
                 keepStandardInOpen: false,
-                cancellationToken: cancellationToken);
+                cancellationToken: cancellationToken,
```

Comment:
cancellationToken should always be the last parameter for Async method. :)

Reply:
Fixed